src/test/kotlin/uk/gov/justice/hmpps/offenderevents/e2e/HMPPSDomainEventsTest.kt

```diff
@@ -304,89 +304,177 @@ class HMPPSDomainEventsTest : QueueListenerIntegrationTest() {
 
   @Nested
   internal inner class MergePrisoner {
-    @BeforeEach
-    fun setUp() {
-      sendToTopic(
-        "BOOKING_NUMBER-CHANGED",
-        """
-                {
-                    "eventType":"BOOKING_NUMBER-CHANGED",
-                    "eventDatetime":"2021-02-08T14:41:11.526762",
-                    "bookingId":1201234,
-                    "previousBookingNumber": "38430A",
-                    "nomisEventType":"BOOK_UPD_OASYS"
-                    }
-                
-                """
-          .trimIndent(),
-      )
-    }
 
     @Nested
-    internal inner class WhenIsReportedAsTransfer {
+    inner class TypeIsBookNumberChanged {
       @BeforeEach
       fun setUp() {
-        PrisonApiExtension.server.stubBasicPrisonerDetails("A5194DY", 1201234L)
-        PrisonApiExtension.server.stubPrisonerIdentifiers("A5694DR", 1201234L)
+        sendToTopic(
+          "BOOKING_NUMBER-CHANGED",
+          // language=JSON
+          """
+
+            {
+              "eventType":"BOOKING_NUMBER-CHANGED",
+              "type":"BOOK_NUMBER_CHANGE",
+              "bookingId":"2952268",
+              "nomisEventType":"P1_RESULT",
+              "eventDatetime":"2024-08-30T15:14:06.0000000Z",
+              "offenderId":"5109860",
+              "bookingNumber":"11141F",
+              "previousBookingNumber":"31226G"
+            }
+           """
+            .trimIndent(),
+        )
+        // send control message so we can check messages have been processed
+        sendToTopic(
```

Comment:
I'm going to do another PR to improve these tests; trying to assert the absence of a event is hard but I have a plan on a better way to do this without control messages and whatnot